scripts/spack/packages/mfem/package.py

```diff
@@ -164,6 +164,55 @@ class Mfem(Package, CudaPackage, ROCmPackage):
     variant('miniapps', default=False,
             description='Build and install miniapps')
 
+    # Custom variant not yet in upstream Spack; see
```

Comment:
Any custom changes on our end need to be marked like this:

```
// SERAC EDIT START - <reason>
whatever change
// SERAC EDIT END
```
This makes it possible for us in the future to know whether changes in their package need to be moved forward when we update.

Reply:
From looking at [other parts of `scripts/spack/packages/mfem/package.py`](https://github.com/LLNL/serac/blob/affb5793a6421abdde9fd29e5892e29c96df84bd/scripts/spack/packages/mfem/package.py#L52), would

```python
# SERAC EDIT BEGIN - reason
pass # <--- changes would go here
# SERAC EDIT END
```

work also?

Reply:
Yes please use the proper comment syntax given that this is a python file. I just quickly typed that up w/o thinking.

Reply:
Added to MFEM package in [`f4ae4e62`](https://github.com/LLNL/serac/pull/699/commits/f4ae4e62b5bb0e944ab92d352f4fedde82f39e14) and to Sundials package in [`008b116f`](https://github.com/LLNL/serac/pull/699/commits/008b116f0ff7894596f10747082a0e77febfbbf1).